lib/common/FileUtils.cpp

```diff
@@ -32,6 +38,7 @@ string FileUtils::ReadFileToString(const string &path)
 std::string FileUtils::Basename(const std::string &path)
 {
 #ifdef WIN32
+#error TODO
```

Comment:
What's this? Looks like windows is not supported.

Note: FileUtils replicates a lot of utility functions in CFuncs. I'm OK with moving the file related stuff from CFuncs to FileUtils, but we shouldn't duplicate the functionality